composer.json

```diff
@@ -1,7 +1,6 @@
 {
     "name": "gowork/dqo",
     "description": "Database Query Objects",
-    "version": "0.1.0",
```

Comment:
:smile:

Reply:
https://blog.packagist.com/tagged-a-new-release-for-composer-and-it-wont-show-up-on-packagist/ :shrug: